### PR TITLE
feat: Implement GET request for Cards List

### PR DESCRIPTION
**Pull request by Pedrael**

### CODE REVIEW

File: src/components/CardsList.tsx [hunk 1]
```diff
@@ -20,18 +23,28 @@ export function CardsList({ cards }: CardsListProps) {
...
+            position={position}
             email={email}
             phone={phone}
+            photo={photo}
           />
         ))}
       </Box>
-      <CustomisedButton>Show more</CustomisedButton>
+      <Box display="flex" alignItems="center">
+        <CustomisedButton onClick={onClick}>Show more</CustomisedButton>
```
Comment (ReidenXerx) [inline src/components/CardsList.tsx:39]: Naming

File: src/types.ts [hunk 2]
```diff
@@ -1,8 +1,10 @@
 export type InfoCard = {
   id: string
-  imgURL?: string
-  title?: string
-  description?: string
+  name?: string
```
Comment (ReidenXerx) [inline src/types.ts]: Is really all of these props is optional?

**Review by ReidenXerx — CHANGES_REQUESTED** [review]

File: src/components/CardsList.tsx (hunk 1 above)
Comment (Pedrael) [reply src/components/CardsList.tsx:39]: will be fixed in the next pr

File: src/types.ts (hunk 2 above)
Comment (Pedrael) [reply src/types.ts]: fixed

**Review by ReidenXerx — APPROVED** [review]

**Pull request merged**